src/azure-cli/azure/cli/command_modules/acr/import.py

```diff
@@ -94,18 +95,23 @@ def acr_import(cmd,
                                               target_tags=target_tags,
                                               untagged_target_repositories=repository,
                                               mode=ImportMode.force.value if force else ImportMode.no_force.value)
+
     result_poller = client.import_image(
         resource_group_name=resource_group_name,
         registry_name=registry_name,
         parameters=import_parameters)
 
+    if no_wait:
+        return result_poller
+
     return _handle_result(cmd, result_poller, source_registry, source_image, registry)
 
 
 def _handle_result(cmd, result_poller, source_registry, source_image, registry):
     from msrestazure.azure_exceptions import ClientException
     try:
-        result = LongRunningOperation(cmd.cli_ctx, 'Importing image...')(result_poller)
```

Comment:
Inline with my comment above, we might need to rethink how we approach handle result when we have a wait versus a no wait. There are a few examples of calls to sdk_no_wait here in this [file](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/vm/custom.py#L1013).